test/evp_kdf_test.c

```diff
@@ -1260,6 +1260,16 @@ static int test_kdf_sshkdf(void)
     return ret;
 }
 
+static int test_kdfs_same( EVP_KDF *kdf1, EVP_KDF *kdf2)
+{
+    /* Fast path in case the really are the same algorithm pointer */
```

Comment:
Maybe remove this first comment - or make it make sense :)

Reply:
It reads badly now, it didn't at the time.  Regardless, fixed.